README.md

```diff
@@ -1,2 +1,7 @@
-# Projecto-XSZ
-Projecto personal para la asignatura de IV
+## Problema:
+Los gimnasios suelen tener para cada semana, un horario de actividades (yoga, clases de boxeo, etc) que se impartirán. Pero para planificar dicho horario, es necesario tener en cuenta la disponibilidad de los profesores, la hora con más afluencia, el tipo de actividad en concreto, la duración, etc. Además, interesa saber la popularidad de una actividad tras su impartición para las futuras organizaciones de actividades.
```

Comment:
¿Tienes conocimiento personal de este problema? ¿Realmente tienen problemas los gimnasios para asignar profesores? ¿Quién tiene ese problema?
También, *rule of thumb*: si en la descripción de *un* problema metes un *además* es que no estás entendiendo bien el problema. O son dos problemas, o simplemente las dos cosas vienen de un problema común (que no estás identificando).

Reply:
Sé que son dos problemas: uno la valoración de las actividades y otro la planificación de esas actividades. Pero creo que es interesante juntarlas porque la primera repercute en la segunda.

Reply:
Para esta asignatura es conveniente que te fijes en *un* problema y programes en diferentes etapas la lógica de negocio necesaria para resolverla.